compiler/src/iree/compiler/Codegen/Common/OptimizeVectorTransferPass.cpp

```diff
@@ -19,10 +19,20 @@
 #include "mlir/Transforms/GreedyPatternRewriteDriver.h"
 #include "mlir/Transforms/LoopInvariantCodeMotionUtils.h"
 
+#define DEBUG_TYPE "iree-codegen-optimize-vector-transfer"
+
 namespace mlir {
 namespace iree_compiler {
 namespace {
 
+static void debugPrint(func::FuncOp funcOp, const char *message) {
+  LLVM_DEBUG({
+    llvm::dbgs() << "//--- " << message << " ---//\n";
+    funcOp.print(llvm::dbgs(), OpPrintingFlags().useLocalScope());
+    llvm::dbgs() << "\n\n";
+  });
+}
+
```

Comment:
Can we follow the existing style that uses MACRO instead of creating a method? E.g.,

```
#define DBGS() (llvm::dbgs() << "[" DEBUG_TYPE "]: ")
#define LDBG(X) LLVM_DEBUG(DBGS() << X << "\n")

...

LDBG("..." << funcOp);
```